CHANGELOG.md

```diff
@@ -61,6 +57,7 @@
 - Remove empty page tutorials/intelmq-manager (PR#2562 by Sebastian Wagner).
 
 ### Packaging
+- Packages for Ubuntu 24.04 (by Sebastian Wagner, fixes #2571).
```

Comment:
Again, link to any instructions needed? Are there migration instructions needed for Ubuntu 22.x -> 24.04 ?

Reply:
Do we want to document how to do a distribution upgrade?
FWIW: https://documentation.ubuntu.com/server/how-to/software/upgrade-your-release/index.html

Reply:
Yes! We want to *link* to what people should read. Linking is good.

Reply:
Added the link to the news file